packages/transport-bridge/src/http.ts

```diff
@@ -36,11 +36,12 @@ const validateDescriptorsJSON: RequestHandler<JSON, Descriptor[]> = (request, re
 };
 
 const validateAcquireParams: ParamsValidatorHandler<{
-    path: string;
+    path: PathPublic;
     previous: Session | 'null';
 }> = (request, response, next) => {
     if (
         typeof request.params.path === 'string' &&
+        Number.parseInt(request.params.path, 10) &&
```

Comment:
Probably not a big deal, but this allows negative numbers or even hex numbers and disallows zero. The `previous` check below is much more strict (`\d+`).